digdag-ui/console.jsx

```diff
@@ -1326,7 +1372,7 @@ class TaskListView extends React.Component {
           </thead>
           <tbody>
             {
-              Array.from(this.props.tasks.values()).map(task =>
+              sortTasksForTreeView(Array.from(this.props.tasks.values())).map(task =>
```

Comment:
According to MDN, [Map.prototype.values()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/values) returns Iterator. And thus flow shows this error without the cast:

```
1375:               sortTasksForTreeView(this.props.tasks.values()).map(task =>
                                         ^^^^^^^^^^^^^^^^^^^^^^^^^ $Iterator. This type is incompatible with the expected param type of
1304: function sortTasksForTreeView (tasks: Array<Task>): Array<Task> {
                                            ^^^^^^^^^^^ array type
```